packages/orbit-components/src/InputSelect/__tests__/index.test.tsx

```diff
@@ -216,6 +216,46 @@ describe("InputSelect", () => {
     expect(screen.getAllByRole("option")).toHaveLength(1 + 2 + 1 + 4); // (1 previously selected, 2 asian, 1 american, 4 all)
   });
 
+  it("can be readOnly", async () => {
+    const onFocus = jest.fn();
+
+    render(
+      <InputSelect
+        id={id}
+        label={label}
+        options={options}
+        name={name}
+        defaultSelected={jetLiOption}
+        onFocus={onFocus}
+        readOnly
+      />,
+    );
+
+    const input = screen.getByRole("textbox");
+    const dropdown = screen.queryByRole("listbox");
+
+    expect(input).toHaveValue(jetLiOption.title);
+
+    await user.tab();
+    expect(onFocus).toHaveBeenCalled();
```

Comment:
I call this assertion for every of event (fireEvent, user.tab, user.type)

Reply:
I removed tests for events "Enter" and "onKeyDown". I think they can't be triggered from the mobile view (?).